README.md

```diff
@@ -18,20 +18,12 @@ data to the framework.
 ## Usage
 
 ```yml
-...
 - name: Upload test report
   uses: Brightspace/test-reporting-action@main
   with:
-    aws-access-key-id: ${{secrets.AWS_ACCESS_KEY_ID}} # required
-    aws-secret-access-key: ${{secrets.AWS_SECRET_ACCESS_KEY}} # required
-    aws-session-token: ${{secrets.AWS_SESSION_TOKEN}} # required
-    report-path: ./d2l-test-report.json # optional
-    lms-build-number: '20.24.1.123456' # optional
```

Comment:
the comment for these fields in the INputs section is a bit weird: `Will result in failure if already present in report.`
so, what happens if it's _not_ in the report and i don't provide it?

if we don't want to examine the implementation further, something like "will be set to X if not present in the report. will throw an error if provided and already present" should do the trick

Reply:
If it's not in the report and you don't provide it, nothing happens. It just doesn't get sent. The only reason it fails if in the report and also include it here is that the report one is probably the right one.

Reply:
ok, a little nitpicky maybe, but failure & error are different things, so I still think the latter part of my suggestion is valid... `will throw an error if provided and already present` would make it clearer 🤷‍♂️

Reply:
edit: just looked at the code and it looks like we don't actually throw an error here? so.... i take it something downstream will explode?  if so, that feels weird....
or am i just reading this all backwards? 🙃

Reply:
I'm pretty sure it throws up somewhere. Let me find that.

Reply:
edit#2: ok i found it now :P

Reply:
https://github.com/Brightspace/test-reporting-action/blob/main/src/report.js#L218-L240

Reply:
This is during the finalizing of the report to send to AWS.